styleguide/source/_patterns/01-molecules/12-ec/02-ec-bundle-card/_ec-bundle-card.scss

```diff
@@ -0,0 +1,101 @@
+.ec-bundle-card {
```

Comment:
The way this class and its files in PatternLab is named doesn't follow our current naming conventions. Since the conventions are loose anyway, and some reorganizing of cards/where things are stored should happen hopefully in the near future, I'm fine with merging as-is for now.